layers/meta-resin-allwinner/recipes-core/packagegroups/packagegroup-resin-connectivity.bbappend

```diff
@@ -1 +1 @@
-CONNECTIVITY_FIRMWARES_append = " linux-firmware-ap6212 linux-firmware-brcm43362"
```

Comment:
Why delete these firmwares?

Reply:
Our currently supported machines BananaPiM1+ and OrangePi Plus2 don't use these firmwares, but it's not a bad idea to keep them